### ISSUE TITLE
v1.5.0 release is broken

### ISSUE THREAD

**Issue by Peder2911** [DavidVujic/python-polylith#193]
**Describe the bug**
The latest version of `polylith-cli` released on Github has a syntax error.

**To Reproduce**
Download the latest source release, check line 16 in `polylith_cli/polylith/check/report.py`.

Does this project use CI/CD? If not, I would strongly encourage doing so to make releases more dependable. Due to the difference between the code in the repo and the code on Pypi I can only assume that this latest release was a mistake of some sort, but there is no audit trail showing how the release was built and uploaded.

**Comment by Peder2911** [comment]
The offending f-string is only compatible with Python 3.12, which means that the issue is that the release is only compatible with Python >=3.12. This, however, is not indicated on Pypi.

**Comment by DavidVujic** [comment]
Oh, that is not good. Thank you for reporting @Peder2911!

I have it setup in a CI with Python 3.8 (linting, testing, mypy) with manual releasing, but must have done something wrong.

Is it this one? https://github.com/DavidVujic/python-polylith/blob/main/components/polylith/check/report.py#L23C17-L23C100

**Comment by DavidVujic** [comment]
I can confirm this is a bug, this part of the code isn't touched by tests and slipped through the checks. I will fix this.

**Comment by Peder2911** [comment]
Yes that's the line, but it's not the same in the code released on Pypi. If you download the tar from Pypi you'll see the 3.12-exclusive f-string in that same module.

No worries David, thanks for fixing!

**Comment by DavidVujic** [comment]
I'll do that! I think the line difference could be because of the adding of a custom top namespace, that I use when deploying different installable libraries from the same Polylith Workspace. That part is done during build of a wheel/sdist.

**Comment by DavidVujic** [comment]
I have released a new version of the CLI and the Poetry plugin that hopefully fixes this issue. Thank you again for reporting this error, much appreciated!
